docs/about.md

```diff
@@ -0,0 +1 @@
+This is the about page.

```

Comment:
Any idea why this page is there ? I can't see where it is being referenced: can it just be removed ?